src/main/scala/coupledL2/tl2chi/MSHR.scala

```diff
@@ -334,6 +334,7 @@ class MSHR(implicit p: Parameters) extends TL2CHIL2Module with HasCHIOpcodes {
       (release_valid2 && isWriteBackFull)                -> WriteBackFull,
       (release_valid2 && !isWriteBackFull)               -> Evict,
       (req.opcode === AcquirePerm && req.param === NtoT) -> MakeUnique,
+      (req.opcode === AcquirePerm && req.param === BtoT) -> MakeUnique,
```

Comment:
No need to distinguish different `req.param`. `req.opcode === AcquirePerm -> MakeUnique` is okay.

Reply:
Got it!